telemetry/definitions/commonDefinitions.json

```
@@ -773,13 +773,61 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "numberOfConnections",
+            "type": "int",
+            "description": "Number of Existing connections the user has."
+        },
+        {
+            "name": "builderIdStatus",
+            "type": "string",
+            "description": "login status with builderId option",
+            "allowedValues": [
+                "not_attempted",
+                "attempted",
+                "canceled", 
+                "failed"
+            ]
+        },
+        {
+            "name": "tab",
```

- this is a very generic name and unfortunately the telemetry definitions don't have namespacing or nesting, so we need to use common prefixes to help with that.
- "tab" is pretty coupled with the current UI, maybe "component" or "element" would be more appropriate?

```suggestion
            "name": "authUIElement",
```